geomstats/geometry/hyperbolic.py

```diff
@@ -833,3 +867,32 @@ def dist(self, point_a, point_b):
         else:
             raise NotImplementedError(
                 'dist is only implemented for ball and extrinsic')
+
+    def retraction(self, tan_vector, base_point):
+        """Poincaré ball model retraction.
+
+           Approximate the exponential map of hyperbolic space,
+           currently working only with poincare ball.
+           The retraction used is the one proposed in "Poincaré
```

Comment:
Use the convention for style for references in docstrings (see code in stiefel.py for example).

It will allow this reference to be nicely formatted in the automatically generated documentation website.